src/lib/autoinstall/clients/inst_autosetup_upgrade.rb

```diff
@@ -226,6 +227,23 @@ def main
           false
         )
 
+        #
+        # Checking if at least one base product has been selected/evaluated.
+        #
+        begin
+          Product.FindBaseProducts
+        rescue StandardError
+          msg = _("No new base product has been set.\n" \
+           "Please check the <b>software</b>/<b>products</b> entry in the " \
```

Comment:
Maybe a bit more generic message (without the AY profile it can suggest creating it with the required content).

```suggestion
           "It can be specified in the <b>software</b>/<b>products</b> entry in the " \
```